bin/create_user

```diff
@@ -1,16 +1,19 @@
 #!/usr/bin/env ruby
 
-require 'digest'
 require 'mongo'
+require 'cherry_silt'
 
-mongo_client = Mongo::Client.new([ '127.0.0.1:27017' ], database: 'mud')
+client = Mongo::Client.new(['127.0.0.1:27017'], database: 'mud')
+
+client[:player].drop
```

Comment:
Already merged but, why are we dropping the player table?  Just to clear it for the tests?